config/stage.json

```diff
@@ -72,6 +72,7 @@
 		"post-editor/html-toolbar": true,
 		"post-editor/insert-menu": true,
 		"post-editor/image-editor": true,
+		"post-editor/mentions": true,
```

Comment:
So would we just remove the feature flag from `stage.json` and `production.json`? I assume if they're  not explicitly set to `true` the feature won't get loaded if it sits behind a config check?